tests/utils.py

```diff
@@ -12,6 +12,9 @@
 import pathspec  # type: ignore[import]
 import tomli
 
+# Used to install system-wide packages for different OS types:
+metadata_mapping = {"linux": "apt_dependencies", "darwin": "brew_dependencies", "win32": "choco_dependencies"}
```

Comment:
Nit: maybe use SCREAM_CASE for this, if it's a constant imported from multiple files?

```suggestion
METADATA_MAPPING = {"linux": "apt_dependencies", "darwin": "brew_dependencies", "win32": "choco_dependencies"}
```